examples/graph_sage_unsup.py

```diff
@@ -78,17 +63,16 @@ def train():
     model.train()
 
     total_loss = 0
-    for batch_size, n_id, adjs in train_loader:
-        # `adjs` holds a list of `(edge_index, e_id, size)` tuples.
-        adjs = [adj.to(device) for adj in adjs]
-        optimizer.zero_grad()
+    for loader in train_loader:
 
-        out = model(x[n_id], adjs)
-        out, pos_out, neg_out = out.split(out.size(0) // 3, dim=0)
+        optimizer.zero_grad()
+        h = model(loader)
+        src_nodes = loader.edge_label_index[0]
+        dst_nodes = loader.edge_label_index[1]
+        out = h[src_nodes] * h[dst_nodes]
+        loss = F.binary_cross_entropy_with_logits(out.sum(dim=-1),
+                                                  loader.edge_label)
 
-        pos_loss = F.logsigmoid((out * pos_out).sum(-1)).mean()
-        neg_loss = F.logsigmoid(-(out * neg_out).sum(-1)).mean()
-        loss = -pos_loss - neg_loss
```

Comment:
is the new loss equivalent to the previous loss.

Reply:
Yes :)